library/Icingadb/Widget/Detail/HostMetaInfo.php

```diff
@@ -31,15 +31,15 @@ public function __construct(Host $host)
     protected function assemble()
     {
         $this->addHtml(
-            new VerticalKeyValue('host_name', $this->host->name),
+            new VerticalKeyValue('host.name', $this->host->name),
             new HtmlElement(
                 'div',
                 null,
-                new HorizontalKeyValue('host_address', $this->host->address ?: '-'),
-                new HorizontalKeyValue('host_address_v6', $this->host->address6 ?: '-')
+                new HorizontalKeyValue('host.address', $this->host->address ?: '-'),
+                new HorizontalKeyValue('host.address6', $this->host->address6 ?: '-')
             ),
             new VerticalKeyValue(
-                'last_state_change',
+                'host.state.last_state_change',
```

Comment:
We should leave the relation path out here, the column name explains enough that it's not strictly required. Otherwise the host name is cut if the v6 address is very long:

![Screenshot from 2021-08-19 08-29-58](https://user-images.githubusercontent.com/16668527/130019256-ecc04633-208f-40ce-990f-f429783020a1.png)
![Screenshot from 2021-08-19 08-29-42](https://user-images.githubusercontent.com/16668527/130019260-b6f8a29f-0c1c-4415-a0f9-ff6baeef28b5.png)